rep-2005.rst

```diff
@@ -39,20 +39,53 @@ The list below is curated to include repositories that are:
 
 We aim that all of these repositories are also:
 
-* actively maintained and following accepted software development processes, including code review and testing.
+* actively maintained and following accepted software development processes, including code review and testing; and
+* at Quality Level 3 or better, as defined in REP 2004; and
+* are maintained by an organization or at least two individuals.
+
+The list should be self-contained and exhaustive: if a package appears on the list, then all of its ROS 2 dependencies must also appear on the list (third-party or system dependencies should not be included).
```

Comment:
This might not be real problem, but I feel this requires some sort of dependency crawler tool to verify that this list stays accurate. I don't think we can easily supervisor this exhaustive list. Do we have something like this already along the lines of the buildfarm?